Tests/ApolloTests/StarWars/API.swift

```diff
@@ -54,59 +62,68 @@ public final class CreateReviewForEpisodeMutation: GraphQLMutation {
   public struct Data: GraphQLMappable {
     public let createReview: CreateReview?
 
-    public init(reader: GraphQLResultReader) throws {
-      createReview = try reader.optionalValue(for: Field(responseName: "createReview", arguments: ["episode": reader.variables["episode"], "review": reader.variables["review"]]))
+    public init(values: [Any?]) {
+      createReview = values[0] as! CreateReview?
     }
 
     public struct CreateReview: GraphQLMappable {
       public let __typename: String
       public let stars: Int
       public let commentary: String?
 
-      public init(reader: GraphQLResultReader) throws {
-        __typename = try reader.value(for: Field(responseName: "__typename"))
-        stars = try reader.value(for: Field(responseName: "stars"))
-        commentary = try reader.optionalValue(for: Field(responseName: "commentary"))
+      public init(values: [Any?]) {
+        __typename = values[0] as! String
```

Comment:
Why did values change to arrays vs. dynamic dictionaries? I’m also curious how this would work for a query like:

```graphql
{
  hello
  world
  hello
  alias: world
}
```

Or:

```graphql
{
  foo {
    a b
    bar {
      d
    }
  }
  foo {
    c
    bar {
      e
      f
    }
  }
}
```